circle.yml

```diff
@@ -1,17 +1,21 @@
-machine:
-  node:
-    version: 5.1.0
-
 dependencies:
   override:
     - curl -L https://atom.io/download/deb -o atom-amd64.deb
     - sudo dpkg --install atom-amd64.deb || true
     - sudo apt-get update
     - sudo apt-get -f install
-    - nvm use "v5.1.0"
-    - npm install --development
+    - node --version
+    - npm --version
+    - atom --version
+    - npm prune
+    - npm install
+    - apm rebuild
```

Comment:
It rebuilds any native node_modules with the correct version of NaN and stuff. It's required to have flat dependency structure of npm@3 (to make eslint work) while still making sure the native modules are compiled with apm